handlers/web/categoriesWebHandler.js

```diff
@@ -23,6 +23,7 @@ CategoriesWebHandler.technologiesForCategory = function (req, res) {
     technology.getAllForCategory(cname.toLowerCase(), function (values) {
 
         if (values == null) {
```

Comment:
Adding a length check idd seems usefull. I would leave the null check though - just in case if anyone ever changes the dao - we would have errors then. Does that sound good?